### ISSUE TITLE
Test on Linux

### ISSUE THREAD

**Issue by mwermelinger** [dsa-ou/m269-installer#1]
The instructions need to be tested and possibly modified for Linux systems. As I don't have Linux, this requires input from others.

1. Please follow the [software installation instructions](https://dsa-ou.github.io/m269-installer) and make a note of anything that is unclear, missing, unexpected (behaved differently from what the instructions say) or not working, as you install the software on your Linux system. 
2. After installing the software (Section 2 of the instructions), download the book from the M269 site into the M269 folder you created. Then proceed with Section 3 of the instructions to check the software is working. Note anything that is unclear or not working.
3. Post a comment below, indicating what Linux system you used and whether everything went fine or what problems you encountered.

Many thanks!

**Comment by densnow** [comment]
Everything seems to be working fine in terms of installation, adding aliases to the config and all of that. I just have a very minor issue with the new `nb` alias pushing the `jupyter` process into the background.

When the process is pushed into the background  it cannot be stopped using the normal double Ctrl+c, and can only be killed using the "quit" in the browser (or a command like `pkill`). When I click  "quit" in the browser, it kills the process, but leaves the terminal in a state where the command prompt is not restored (it requires a further Ctrl+c to get it back). This is not really a big deal, but it does not quite line up with the "using the software" instructions. It might be helpful to mention that after "quitting," a single Ctrl+c is required to regain the terminal prompt. Alternatively, not pushing the process into the background would allow both double Ctrl+c and browser "quitting" to kill the process and restore the prompt.

**Comment by mwermelinger** [comment]
Thanks for checking. Are you sure you need Ctrl-C? When I type `nb` I immediately get back to the prompt, because the process is in the background. After a few seconds I get in the terminal all the messages from Jupyter starting, but though it seems that Jupyter is now hogging the terminal, you can actually type any command (like `allowed`) or jut ENTER to get the prompt back.

**Comment by densnow** [comment]
@mwermelinger, I Just gave the new changes and instructions from 9a27fd9984aa3bd2b5cef0940e43e78ab8df323b a "test drive". Again, it seems all is working as expected in terms of the script behavior, and I really like the simplified installation process, refined script and instructions!

The one thing I noticed this time (when using `curl` to download script) was that `install-m269.sh` needed to be  given executable permissions before it could be run. I must admit last time I tested I had cloned the repo, so this could have been the case when downloading the zip file too! Maybe an instruction for a `chmod +x install-m269.sh` might be needed?
